.nvmrc

```diff
@@ -1 +1 @@
-8.9.4
+8.11.3
```

Comment:
So we need to change the version of NVM?

Reply:
No, this changes the version of node. I needed it in order to update a dependency.